apps/hpc-cdm/src/app/components/target-access-management.tsx

```diff
@@ -173,6 +173,7 @@ export const TargetAccessManagement = (props: Props) => {
               invites.map((item, i) => {
                 return (
                   <C.ListItem
+                    key={i}
```

Comment:
This is a dangerous thing to do. Please read this "Pitfall" section in [new React docs](https://react.dev/learn/rendering-lists#why-does-react-need-keys) (old docs warn about using index as key too):
> You might be tempted to use an item’s index in the array as its key. In fact, that’s what React will use if you don’t specify a `key` at all. But the order in which you render items will change over time if an item is inserted, deleted, or if the array gets reordered. Index as a key often leads to subtle and confusing bugs.

In [old React docs](https://legacy.reactjs.org/docs/lists-and-keys.html#keys), you can find a link to a [good article](https://robinpokorny.com/blog/index-as-a-key-is-an-anti-pattern/) explaining rules of when index is good to be used as a `key`:
> To help you decide, I put together three conditions which these examples have in common:
> 1. the list and items are static–they are not computed and do not change;
> 2. the items in the list have no ids;
> 3. the list is never reordered or filtered.
>
> When all of them are met, you **may safely use the index as a key**.

In fact, this component already has some unsafe usages of indexes as keys, so I am going to cover all of them, not only this which you have added:
- List of Active users: [Here](https://github.com/UN-OCHA/hpc-cdm/blob/b196669f1031b936270ab7d7d694dc39030e3b28/apps/hpc-cdm/src/app/components/target-access-management.tsx#L85), use `key={item.grantee.id}`
- Pending users: Use `key={item.email}`, as we said above, this is the part you're changing with this commit
- Audit log: You missed defining the key here and React complains. [Here](https://github.com/UN-OCHA/hpc-cdm/blob/b196669f1031b936270ab7d7d694dc39030e3b28/apps/hpc-cdm/src/app/components/target-access-management.tsx#L281), use ```key={`${item.actor.id}-${item.grantee.id}-${item.date}`}```